### PR TITLE
OHRM5X-1282:  link validation

### PR DESCRIPTION
**Pull request by Ajiharan007**

### CODE REVIEW

File: src/plugins/orangehrmAuthenticationPlugin/entity/ResetPasswordRequest.php [hunk 1]
```diff
@@ -58,6 +58,13 @@ class ResetPasswordRequest
      */
     private string $resetCode;
 
+    /**
+     * @var bool
+     *
+     * @ORM\Column(name="valid", type="boolean")
+     */
+    private bool $valid;
```
Comment (RajithaKumara) [inline src/plugins/orangehrmAuthenticationPlugin/entity/ResetPasswordRequest.php:66]:
```suggestion
    private bool $expired;
```
any idea to refactor all wording to `expired`

File: installer/Migration/V5_0_0/Migration.php [hunk 2]
```diff
@@ -336,6 +337,20 @@ private function configureMenuItems(string $configName, string $parentMenuTitle)
             ->executeQuery();
     }
 
+
+    /**
+     * @return void
+     */
+    private function addValidColumnToRequestResetPassword(): void
```
Comment (RajithaKumara) [inline installer/Migration/V5_0_0/Migration.php:344]: this is correct. but have to create a new `Migration.php` file under
`installer/Migration/V5_1_0`

**Review by RajithaKumara — CHANGES_REQUESTED** [review]

File: src/plugins/orangehrmAuthenticationPlugin/entity/ResetPasswordRequest.php (hunk 1 above)
Comment (Ajiharan007) [reply src/plugins/orangehrmAuthenticationPlugin/entity/ResetPasswordRequest.php:66]: yes expired is more suitable than valid

File: installer/Util/AppSetupUtility.php [hunk 3]
```diff
@@ -57,6 +57,7 @@ class AppSetupUtility
         '5.0' => [
             \OrangeHRM\Installer\Migration\V5_0_0_beta\Migration::class,
             \OrangeHRM\Installer\Migration\V5_0_0\Migration::class,
+            \OrangeHRM\Installer\Migration\V5_1_0\Migration::class,
```
Comment (Ajiharan007) [inline installer/Util/AppSetupUtility.php:60]: @RajithaKumara this is correct or need to add in **5.1**

**Pull request closed without merging**